modules/swagger-codegen/src/main/resources/JavaJaxRS/resteasy/queryParams.mustache

```diff
@@ -1 +1 @@
-{{#isQueryParam}}{{#useBeanValidation}}{{>beanValidationQueryParams}}{{/useBeanValidation}} @QueryParam("{{baseName}}") {{{dataType}}} {{paramName}}{{/isQueryParam}}
\ No newline at end of file
+{{#isQueryParam}}{{#useBeanValidation}}{{>beanValidationQueryParams}}{{/useBeanValidation}}{{#defaultValue}} @DefaultValue("{{{defaultValue}}}"){{/defaultValue}} @QueryParam("{{baseName}}") {{{dataType}}}{{paramName}}{{/isQueryParam}}
```

Comment:
```
{{{dataType}}}{{paramName}}
```
should be
```
{{{dataType}}} {{paramName}}
```
with a space between.

Reply:
I just committed the change you mention @wing328 .